### PR TITLE
Add pub_publish.sh script and update readme.

### PR DESCRIPTION
**Pull request by jacob314**
use --force to avoid manually confirming each publish operation.

### CODE REVIEW

**Comment by devoncarew** [comment]
You may consider converting this to dart and adding to the repo tool (in `//bin/`).

File: tool/README.md [hunk 1]
```diff
@@ -85,30 +85,9 @@ dart bin/devtools.dart
...
-pushd packages/devtools_app
-pub publish
-
-popd
-pushd packages/devtools
-pub publish
-
-popd
-
+./too/pub_publish.sh
```
Comment (kenzieschmoll) [inline tool/README.md:90]: ./tool ?

**Review by kenzieschmoll — COMMENTED** [review]
one question about the behavior: will this still exit on errors, but just force on warnings?

**Review by kenzieschmoll — APPROVED** [review]

File: tool/README.md (hunk 1 above)
Comment (jacob314) [reply tool/README.md]: done

**Comment by jacob314** [comment]
Writing this as a dart script instead of a shell script might obfuscate what is going on a bit

> one question about the behavior: will this still exit on errors, but just force on warnings?

This will still exit on errors. We get warnings because we have tight version constraints and don't update changelogs for all of these packages.

**Pull request merged**